test/http2_test/http2_base_server.py

```diff
@@ -0,0 +1,173 @@
+import struct
+import messages_pb2
+import logging
+
+from twisted.internet.protocol import Protocol
+from twisted.internet import reactor
+from h2.connection import H2Connection
+from h2.events import RequestReceived, DataReceived, WindowUpdated, RemoteSettingsChanged, PingAcknowledged
+from h2.exceptions import ProtocolError
+
+READ_CHUNK_SIZE = 16384
+GRPC_HEADER_SIZE = 5
```

Comment:
These look like internal constants, and should start with an underscore, _READ_CHUNK_SIZE, _GRPC_HEADER_SIZE.